pronto_mia/lib/core/models/profiles.dart

```diff
@@ -5,10 +5,32 @@ class Profile {
   final AccessControlList accessControlList;
 
   const Profile({this.description, this.accessControlList});
+
+  Profile.fromJson(Map<String, dynamic> json)
+      : description = profiles.entries
+            .firstWhere(
+              (entry) => entry.value.accessControlList
+                  .isEqual(AccessControlList.fromJson(json)),
+              orElse: () => const MapEntry(
+                  'custom', Profile(description: 'Benutzerdefiniert')),
+            )
+            .value
+            .description,
+        accessControlList = AccessControlList.fromJson(json);
 }
 
-class Profiles {
-  static const Profile administrator = Profile(
+final profiles = {

```

Comment:
Shouldn't there be a profile `other` for configurations that do not match a profile?